src/helpers.js

```diff
@@ -17,6 +17,11 @@ Marionette.isNodeAttached = function(el) {
   return Backbone.$.contains(document.documentElement, el);
 };
 
+// Merge `keys` from `options` onto `this`
+Marionette.mergeOptions = function(options, keys) {
+  if (!options) { return; }
+  _.extend(this, _.pick(options, keys));
```

Comment:
We could also override extend and merge events + ui, etc, as well as mergeOptions (though that would be breaking so it would have to come in v3)